packages/dashboard-core-plugins/package.json

```diff
@@ -44,6 +44,7 @@
     "@deephaven/log": "file:../log",
     "@deephaven/redux": "file:../redux",
     "@deephaven/utils": "file:../utils",
+    "@deephaven/storage": "file:../storage",
```

Comment:
You need to run `npm install` and commit the modifications to `package-lock.json` as a result of this change.